handler/authorizationHandler.go

```diff
@@ -74,8 +75,12 @@ func (a AuthorizationHandler) Decorate(delegate http.Handler) http.Handler {
 			WriteJsonError(response, forbiddenStatusCode, message)
 			return
 		}
-
-		valid, err := a.Validator.Validate(token)
+		
+		ctx := *new(context.Context)
```

Comment:
don't create a Context like this.  Per the API docs, use `context.Background()`

https://godoc.org/context#Background